services/core-web/src/reducers/rootReducer.js

```diff
@@ -63,6 +65,7 @@ export const reducerObject = {
   ...reportReducer,
   ...userReducer,
   ...noticeOfWorkReducer,
+  [DOCUMENTS]: documentReducer,
```

Comment:
IMO please either keep this consistent with the other reducer types, or change the others to match this.

Reply:
Done